tests/molecules/He_ae/check_opt.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+from optparse import OptionParser
+import xml.etree.ElementTree as ET
+import sys
+import os
+
+
+def exit_fail(msg=None):
+    if msg!=None:
+        print(msg)
+    #end if
+    print('Test status: fail')
+    exit(1)
+#end def exit_fail
+
+
+def exit_pass(msg=None):
+    if msg!=None:
+        print(msg)
+    #end if
+    print('Test status: pass')
+    exit(0)
+#end def exit_pass
+
+
+# Open the XML file and return the jub_b value
+def get_jub_b(file):
+    tree = ET.parse(file)
+    root = tree.getroot()
+    for elem in root.iterfind('wavefunction/jastrow/correlation/'):
+        return [float(i) for i in elem.text.split()]
+#end def get_jub_b
+
+
+passfail = {True:'pass',False:'fail'}
+def run_opt_test(options):
+
+
+    prefix_file = options.prefix+'.s'+str(options.series).zfill(3)+'.opt.xml'    
+
+    # Set default for the reference
+    ref_file = './qmc-ref/{0}'.format(prefix_file) if options.ref is None else options.ref
+
+    if not os.path.exists(prefix_file):
+       exit_fail("Test not found:" + prefix_file)       
+
+    if not os.path.exists(ref_file):
+       exit_fail("Reference not found:" + ref_file)
+
+    output = get_jub_b(prefix_file)
+    reference = get_jub_b(ref_file)
+
+    if len(output) != len(reference):
+       exit_fail('Number of coefficient in test({0}) does not match with the reference({1})'.format(len(output),len(reference)))
+
+    success = True
+   
+    for i in range(len(output)):
+       quant_success = float(output[i]) == float(reference[i])
```

Comment:
This floating point comparison should have a tolerance.  E.g. abs(float(output[i])-float(reference[i]))<1e-5.

Reply:
@jtkrogel Yes, it's right. I added tolerance (1e-6) for the test, and now script prints tolerance and deviation from the reference.